server/util/path/index.js

```diff
@@ -110,6 +110,10 @@ function getExecutablePath(instance_type) {
  */
 function getMediaPaths() {
   var media_paths = [];
+  // User set media_path is first priority
+  if (media_path) {
```

Comment:
What could however happen is that `MEDIA_PATH= npm start` is called whereas it would become an empty string. This however is so seldom and user-provoked that I wouldn't explicitely catch it